docs/resource-categorization.md

```diff
@@ -440,30 +440,33 @@ mf2c-curl-post https://localhost/api/fog-area -d
 
 ### Troubleshooting
 
-- To get the `Host-IP` related information the resource-categorization module is running another container `alpine:latest` . So it might possible that sometimes we may not be able to get the output value from that container.
+- For those who are using `latest-V2.0.16`; To get the `Host-IP` related information the resource-categorization module is running another container `alpine:latest` . So it might possible that sometimes we may not be able to get the output value from that container.
 - Fix the multiple `device_ip` creation issue for a single agent
 
 ## Change LOG
 
+###latest-V2.0.20 (date 19/September/2019)
```

Comment:
this doesn't exist yet. This commit is about V2.0.19

Reply:
added the version - V2.0.19 (actually it was an intermediate version for testing, which is already available in docker-hub)

Reply:
line 448 needs to go, cause that release is not the target in this PR